lib/src/runner/vm/platform.dart

```diff
@@ -102,12 +108,13 @@ class VMPlatform extends PlatformPlugin {
     } else if (_config.pubServeUrl != null) {
       return _spawnPubServeIsolate(path, message, _config.pubServeUrl);
     } else {
-      return _spawnDataIsolate(path, message);
+      return _spawnDataIsolate(path, message, _packageResolver(path));
     }
   }
 }
 
-Future<Isolate> _spawnDataIsolate(String path, SendPort message) async {
+Future<Isolate> _spawnDataIsolate(String path, SendPort message,
+    {PackageResolver resolver}) async {
```

Comment:
this wouldn't work - it's declaring a named argument but being used as a positional argument

Reply:
Done